drivers/mdio/mdio_stm32_hal.c

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2024 BayLibre, SAS
+ * Copyright (c) 2010-2024 Analog Devices Inc.
```

Comment:
I am wondering where did these 2010 ADI copyrights come from since the mdio API in zephyr was added in 2021? therefore I can't imagine how did this code exist in 2010?

Reply:
done, thanks